lightly/api/bitmask.py

```diff
@@ -173,6 +186,34 @@ def intersection(self, other):
         """
         self.x = _intersection(self.x, other.x)
 
+    def difference(self, other):
+        """Calculates the difference of two bit masks.
+        Examples:
+            >>> mask1 = BitMask.from_bin('0b0111')
+            >>> mask2 = BitMask.from_bin('0b1100')
+            >>> mask1.difference(mask2)
+            >>> # mask1.binstring is '0b0011'
+        """
+        self.x = _difference(self.x, other.x)
+
+    def __sub__(self, other):
+        return BitMask(self.x - other.x)
```

Comment:
Same problem here as in `difference`. Also, please re-use existing function, if you implement difference above then use it here (or at least use the `_difference` function)